engine/apps/public_api/serializers/webhooks.py

```diff
@@ -158,7 +159,11 @@ def validate_preset(self, preset):
         raise serializers.ValidationError(PRESET_VALIDATION_MESSAGE)
 
     def validate(self, data):
-        if self.instance and self.instance.preset:
+        if (
+            self.instance
+            and self.instance.preset
+            and self.instance.preset != WebhookPresetOptions.ADVANCED_PRESET_META_DATA.id
```

Comment:
nit, but just in case, if for some reason the `ADVANCED_WEBHOOK_PRESET` setting is not set (e.g. to disable them), this would explode, right? (there is no `.id`)